OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper;
+using OutOfSchool.Services.Models;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Util.CustomComparers;
+
+namespace OutOfSchool.WebApi.Util
+{
+    public class MappingProfile : Profile
+    {
+        public MappingProfile()
+        {
+            CreateMap<WorkshopDTO, Workshop>()
+                .ForMember(dest => dest.Keywords, opt => opt.MapFrom(src => string.Join('¤', src.Keywords.Distinct())))
```

Comment:
`'¤'` - should be a char constant introduced somewhere in `WorkshopDto` or `Workshop`